### DISCUSSION TITLE
if you want to collaborate

### DISCUSSION THREAD

**Discussion by Leci37** [Leci37/TensorFlow-stocks-prediction-Machine-learning-RealTime#2] [General]
I would be delighted, contact me wherever you want 
mail : leciluis@gmail.com

**Comment by ghost** [comment]
Hi Luis,
I've downloaded the repository, installed the dependencies but not really sure what needs to be executed/edited in order to start the tool.
Can you provide a little more guidance please?

**Reply by Leci37** [reply]
Of course, 
here you have all the deatils https://github.com/Leci37/stocks-prediction-Machine-learning-RealTime-telegram#detailed-start-up

**Comment by nityansuman** [comment]
@Leci37 Please remove me from whatever because I get all these random notifications. It would be of great help. Thanks in advance.

**Comment by datasprings** [comment]
> I would be delighted, contact me wherever you want mail : [leciluis@gmail.com](mailto:leciluis@gmail.com)

Thanks... I emailed you and sent over a cup of coffee! Talk soon.

**Comment by thedavidhanks** [comment]
Hi @Leci37, 

Are you still maintaining the project?  

I really appreciate the effort you've put in here.  I've been following your installation and tutorial instructions, but I've been running into hurdle after hurdle.  I've been installing/upgrading/downgrading packages that were not included in the requirements documents.  I've also been fumbling installing TA-Lib

I had initially rewritten the Tutorial file to work with your v5.py indicators, then I realized that this is missing many indicators available in v3 (which includes all TA-Lib) 

Anyway, I finally got the tutorial working with 294 indicators.  I'll be going through your tutorial to get a better understanding of the project.

My questions to you are,
1) Would you like me to provide an updated requirements.txt & _x.y.z.txt?
2) Have you considered putting this in a Docker container, so that using the project (and building TA-Lib) isn't OS dependent?
